### DISCUSSION TITLE
[質問]: Realmの教科書の内容から、カテゴリ部分を削除して実行しようとしています。

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#144] [Question-original]
### エラーの概要

プロジェクトを実行するとnilに関するエラーが出ます。調べたところnilやアンラップに関するエラーのようです。

### エラーが起こるタイミング

実行直後（buildはされる）

### エラーが起こる場所

〇〇ViewController

### 想定している挙動

一番最初の画面が表示される

### 原因だと思われる箇所(render: swift)

![スクリーンショット 2023-02-22 18 59 42](https://user-images.githubusercontent.com/75009903/220653518-42a6a946-a315-483b-830b-3115a0d7ee88.png)

### 自分で調べた資料１

教科書Realmの1と2

### 自分で調べた資料２

https://qiita.com/nsy_python/items/372e6e3737c67bada9f5

### GitHubのURL

https://github.com/KOTARO1209/ToDoList

### TODOリスト

- [x] GitHubに最新のコードをアップロードしました。
- [ ] できるだけ具体的なタイトルを設定しました。

**Comment by txxxxc** [comment]
1. カテゴリーの部分を削除した関係でデータベースが壊れているので、一旦シミュレーター上からアプリ消してみてくださいー！
2. ItemViewControllerからカテゴリ機能に関連するコード(selectedCategoryなど)が残っているので、自分で考えて削除してみてくださいー！

**Reply by txxxxc** [reply]
>自分で調べた資料１
教科書Realmの1と2

教科書はこちらから提供している教科書になるので、自分で調べた資料には入らないと考えています！
自分で調べた資料を共有してもらっている目的としては、解決したいエラーに対してどれくらい自分で考えたかを測る所にあるので、他の資料を調べてください！
自分で調べた資料に教科書を貼るのは今後控えてもらえると助かります🙏🙏🙏

**Reply by KOTARO1209** [reply]
了解です！やってみます！
調べた資料のやつすいません！了解しました！

**Reply by txxxxc** [reply]
はい！よろしくおねがいします！
